spec/features/users/02_user_signs_in_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+feature "user signs In", %{
+  As unauthenticated user
+  I want to sign in
+  So that I add donut and review the app.
+} do
+  # Acceptance Criteria
+  # [ ] Link to sign in exists in the nav-bar
```

Comment:
This (and the other instances of this happening) is an example of how comments are basically outdated as soon as they are written. This line says that `Link to sign in exists in the nav-bar` is an acceptance criteria that has not yet been completed (as indicated by the missing X between the brackets). This is obviously not true because the tests are below. I'm not going to be a stickler on this right now, but it will make your specs more obviously completed if you put the X in or just take out the brackets